### ISSUE TITLE
Not connected on Electrum-Zclassic v.3.2.4

### ISSUE THREAD

**Issue by andreik77** [ZclassicCommunity/electrum-zclassic#22]
Today 3 Dec 2019 my wallet Electrum-Zclassic v.3.2.4 is unable to connect to the network. What is the problem, is any server available? What happened with the server electrum.zcl.community?

**Comment by asuppini** [comment]
Same for me...
Got only 3 servers:
- electrum.zcl.community and 178.128.27.40 never working
- electrum.zclassic.ch usually good but since yesterday, trying to synchronize with only one node connected...!

**Comment by asuppini** [comment]
Have you had some news andreik77 ?

**Comment by andreik77** [comment]
Yes, I have some news. I cought connection for a while and transferred all my coins to FinexBox. I'm not sure I will be so lucky again, so I will live without ZClassic.

**Comment by asuppini** [comment]
Which server worked for you?

**Comment by andreik77** [comment]
electrum.zclassic.ch

**Comment by asuppini** [comment]
Thank you
I left electrum wallet too ! 
Too many problems with it

**Comment by hTrap** [comment]
Electrum v3.2.5 has been released.
New electrum server, electrum.800crypto.club has been added.

**Comment by ttl99** [comment]
if anyone is interested or need it, the problem with electrum zclassic wallet v3.2.6 with server has been fixed, everyone update their wallet, very good
